src/style.css

```diff
@@ -0,0 +1,112 @@
+/* stylelint-disable string-quotes */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-bold";
+  font-style: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+form {
+  margin: 20px 0;
+}
+
+.searh-bar-big {
+  font-family: Avenir;
+  width: 100%;
+  height: 70px;
+  font-weight: 300px;
+  background: #fff;
```

Comment:
default value, you can remove it

Reply:
if I remove it, my search bar seems like input without styles

Reply:
no, I disabled this style and nothing changed 
Please fix it
![image](https://user-images.githubusercontent.com/70009205/232832837-fc800f55-6a17-42ce-acfd-83952fd0cd08.png)